CMakeLists.txt

```diff
@@ -1,11 +1,16 @@
 # Check environment
 cmake_minimum_required(VERSION 3.5)
 
-# Set the project name and version
+# set the project name and version
 project(NEXUS
         VERSION 0.2.0
         LANGUAGES Fortran)
 
+# set compiler
+if(NOT DEFINED ENV{FC})
+  message(FATAL_ERROR "CMAKE_Fortran_COMPILER is not defined")
+endif()
```

Comment:
It doesn't need to be fatal error. I was experimenting with these warning messages, but the behavior seems to be not always as expected, due to cmake's caching and such I guess.
```suggestion
# FC should be used to set compiler
if(NOT DEFINED ENV{FC})
  message(WARNING "FC is not set. You may want to use it to specify a specific compiler.")
else()
  if("$ENV{FC}" STREQUAL "")
    message(WARNING "FC is set to empty string. You may want to use it to specify a specific compiler.")
  else()
    message(STATUS "FC is set: '${FC}'")
  endif()
endif()
```